beta/google-cloud-run/README.md

```diff
@@ -79,9 +79,9 @@ The Cloud Run service for SCIM Bridge will be configured to mount volume using a
     ) --role=roles/secretmanager.secretAccessor
     ```
 
-## Step 3: Deploy your SCIM Bridge
+## Step 3: Deploy your SCIM bridge
 
-Run this command to stream [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) Cloud Run service YAML from this repository, use it to deploy SCIM Bridge inline, and enable public ingress for your SCIM Bridge so that you and your identity provider can connect to its public endpoint:
+Run this command to stream [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) Cloud Run service YAML from this repository, use it to deploy SCIM bridge inline, and enable public ingress for your SCIM bridge so that you and your identity provider can connect to its public endpoint:
```

Comment:
```suggestion
Run this command to stream [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) Cloud Run service YAML from this repository, use it to deploy 1Password SCIM Bridge inline, and enable public ingress for your SCIM bridge so that you and your identity provider can connect to its public endpoint:
```